base_classes/nyaml/NXsample_component.yaml

```diff
@@ -0,0 +1,148 @@
+  unit_cell_abc(NX_FLOAT):
+    deprecated: |
+      Use `NXsingle_crystal` and `NXunit_cell` to describe single crystalline materials.
+    unit: NX_LENGTH
+    doc: |
+      Crystallography unit cell parameters a, b, and c
+    dimensions:
+      dim: [[1, 3]]
+  unit_cell_alphabetagamma(NX_FLOAT):
+    deprecated: |
+      Use `NXsingle_crystal` and `NXunit_cell` to describe single crystalline materials.
+    unit: NX_ANGLE
+    doc: |
+      Crystallography unit cell parameters alpha, beta, and gamma
+    dimensions:
+      dim: [[1, 3]]
+  unit_cell_volume(NX_FLOAT):
+    deprecated: |
+      Use `NXsingle_crystal` and `NXunit_cell` to describe single crystalline materials.
+    unit: NX_VOLUME
+    doc: |
+      Volume of the unit cell
+  sample_orientation(NX_FLOAT):
+    deprecated: |
+      Use `NXsingle_crystal` and `NXunit_cell` to describe single crystalline materials.
+    unit: NX_ANGLE
+    doc: |
+      This will follow the Busing and Levy convention from Acta.Crysta v22, p457 (1967)
+    dimensions:
+      rank: 1
+      dim: [[1, 3]]
+  orientation_matrix(NX_FLOAT):
+    deprecated: |
+      Use `NXsingle_crystal` and `NXunit_cell` to describe single crystalline materials.
+    doc: |
+      Orientation matrix of single crystal sample component.
+      This will follow the Busing and Levy convention from Acta.Crysta v22, p457 (1967)
+    dimensions:
+      rank: 2
+      dim: [[1, 3], [2, 3]]
+  mass(NX_FLOAT):
+    unit: NX_MASS
+    doc: |
+      Mass of sample component
+  density(NX_FLOAT):
+    unit: NX_MASS_DENSITY
+    doc: |
+      Density of sample component
+  relative_molecular_mass(NX_FLOAT):
+    unit: NX_MASS
+    doc: |
+      Relative Molecular Mass of sample component
+  description:
+    doc: |
+      Description of the sample component
+  volume_fraction(NX_FLOAT):
+    deprecated: |
+      Use `NXsample_component_set` for the configuration of the multiple components.
+    doc: |
+      Volume fraction of component
+  scattering_length_density(NX_FLOAT):
+    deprecated: |
+      Use `NXsample_component_set` for the configuration of the multiple components.  
+    unit: NX_SCATTERING_LENGTH_DENSITY
+    doc: |
+      Scattering length density of component
+  unit_cell_class:
+    deprecated: |
+      Use `NXsingle_crystal` and `NXunit_cell` to describe single crystalline materials.
+    doc: |
+      In case it is all we know and we want to record/document it
+    enumeration: [triclinic, monoclinic, orthorhombic, tetragonal, rhombohedral, hexagonal, cubic]
+  space_group:
+    deprecated: |
+      Use `NXsingle_crystal` and `NXunit_cell` to describe single crystalline materials.
+    doc: |
+      Crystallographic space group
+  point_group:
+    deprecated: |
+      Use `NXsingle_crystal` and `NXunit_cell` to describe single crystalline materials.
+    doc: |
+      Crystallographic point group, deprecated if space_group present
+  transmission(NXdata):
```

Comment:
transmission of which type of radiation?

Reply:
Good question, I suspect that was added as part of optical spectroscopy description, maybe @domna can comment?